community/index.md

```diff
@@ -91,6 +91,7 @@ International chat rooms are available as well:
 * **[Scala Fr](https://discord.gg/w6VUKrrZK3)** (Discord)
 * **[scala/it](https://discord.gg/8wadTgcZVt)** (Discord)
 * **[Scala Jp](https://discord.gg/SaXM8s4Cyr)** (Discord)
+* **[Scala-Poland](http://scala-poland.eu)** (Slack)
```

Comment:
according to @rochala this link should be better - direct invite to Slack https://join.slack.com/t/scala-poland/shared_invite/zt-1jeq834yd-iOTH4U1Gto3YWu_lEVY5oA